sktime/forecasting/pytorchforecasting.py

```diff
@@ -30,6 +30,123 @@ class PytorchForecastingTFT(_PytorchForecastingAdapter):
     validation_to_dataloader_params : Dict[str, Any] (default=None)
         parameters to be passed for `TimeSeriesDataSet.to_dataloader()`
         by default {"train": False}
+    model_path: string (default=None)
+        try to load a existing model without fitting. Calling the fit function is
+        still needed, but no real fitting will be performed.
+    random_log_path: bool (default=False)
+        use random root directory for logging. This parameter is for CI test in
+        Github action, not designed for end users.
+
+    Examples
+    --------
+    >>> # import packages
+    >>> from sktime.forecasting.base import ForecastingHorizon
+    >>> from sktime.forecasting.pytorchforecasting import PytorchForecastingTFT
+    >>> from sktime.utils._testing.hierarchical import _make_hierarchical
+    >>> # generate random data
+    >>> data = _make_hierarchical(
+    ...     hierarchy_levels=(5, 200), max_timepoints=50, min_timepoints=50, n_columns=3
+    ... )
+    >>> # define forecast horizon
+    >>> max_prediction_length = 5
+    >>> fh = ForecastingHorizon(range(1, max_prediction_length + 1), is_relative=True)
+    >>> # split X, y data for train and test
+    >>> l1 = data.index.get_level_values(1).map(lambda x: int(x[3:]))
+    >>> X_train = data.loc[l1 < 190, ["c0", "c1"]]
+    >>> y_train = data.loc[l1 < 190, "c2"].to_frame()
+    >>> X_test = data.loc[l1 >= 180, ["c0", "c1"]]
+    >>> y_test = data.loc[l1 >= 180, "c2"].to_frame()
+    >>> len_levels = len(y_test.index.names)
+    >>> y_test = y_test.groupby(level=list(range(len_levels - 1))).apply(
+    ...     lambda x: x.droplevel(list(range(len_levels - 1))).iloc[:-max_prediction_length]
+    ... )
+    >>> # define the model
+    >>> model = PytorchForecastingTFT(
+    ...     trainer_params={
+    ...         "max_epochs": 5,  # for quick test
+    ...         "limit_train_batches": 10,  # for quick test
+    ...     },
+    ... )
+    >>> # fit and predict
+    >>> model.fit(y=y_train, X=X_train, fh=fh)
```

Comment:
Do we want to add this lengthy output into the example? I would propose to not add such an output and perhaps skip this line using # doctest skip

Reply:
lengthy output is removed and # doctest skip is added